Update English version of custom-widget.md file (#849)
ZupIT/beagle-docs: content/en/ios/customization/custom-widget/custom-widget.md

### SizeThatFits

**`sizeThatFits:`** Method to implement your size logic, used in the custom component class.

```swift
override func sizeThatFits(_ size: CGSize) -> CGSize {
    systemLayoutSizeFitting(size)
}
```

{{% /tab %}}
{{< /tabs >}}

### Step 3: Register Widget.

Finally, we need to register our custom widget on Beagle

> So to **register it on Beagle** just call the register function from Coder (a Beagle public dependency) during the settings process of the Beagle environment.

Reviewer: ```suggestion
To do so, just call the register function from Coder (a Beagle public dependency) during the settings process of the Beagle environment.
```
